test/e2e/storage/csi_volumes.go

```diff
@@ -311,14 +312,14 @@ func initCSIHostpath(f *framework.Framework, config framework.VolumeTestConfig)
 	}
 }
 
-func (h *hostpathCSIDriver) createStorageClassTest(node v1.Node) storageClassTest {
-	return storageClassTest{
-		name:         "csi-hostpath",
-		provisioner:  "csi-hostpath",
-		parameters:   map[string]string{},
-		claimSize:    "1Gi",
-		expectedSize: "1Gi",
-		nodeName:     node.Name,
+func (h *hostpathCSIDriver) createStorageClassTest(node v1.Node) testsuites.StorageClassTest {
+	return testsuites.StorageClassTest{
+		Name:         "csi-hostpath",
+		Provisioner:  "csi-hostpath",
+		Parameters:   map[string]string{},
+		ClaimSize:    "1Gi",
```

Comment:
this is where claim size should be coming from right? xref: that comment i made in `setupResource`

Reply:
https://github.com/kubernetes/kubernetes/pull/69441#discussion_r224188265